src/components/Modal/Modal.module.scss

```diff
@@ -22,4 +23,36 @@
   border: 1px solid #{$grey-dark};
 
   background: #{$almost-black};
+
+  position: relative;
+}
+
+.WiderContent {
```

Comment:
Hey @mrval5 - we can nest these styles to reflect the component tree like so :

```
@import '../../styles/colors';
@import '../../styles/typography';

.Overlay {
  display: grid;
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;

  place-items: center;
  overflow-y: auto;
  z-index: 10000;

  background: rgba(0, 0, 0, 0.9);

  .Content {
    min-width: 18.75rem; // 300px
    padding: 1.875rem; // 30px
    border-radius: 1rem;
    border: 1px solid $grey-dark;

    background: $almost-black;

    position: relative;
  }

  .WiderContent {
    min-width: 37.64rem;

    .Head {
      display: flex;
      justify-content: space-between;
      align-items: center;
      border-bottom: 1px solid $grey-dark;

      margin-bottom: 2.476rem;

      .Title {
        font-size: 1.48rem;
        font-weight: $font-weight-bold;
      }

      .CloseIcon {
        color: inherit;
      }
    }

    .FooterActions {
      display: flex;
      justify-content: space-between;
      align-items: center;

      margin-top: 2.476rem;
    }
  }
}
```

Reply:
We try to follow this structure with styling - also placing classes in order they are in the tree, personally I feel this improves the readability. I've ordered the classes and nested here.

Reply:
Thanks for your review, I've done the changes as suggested